catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -42,6 +42,20 @@ Implementation details
         <argument ref="queryResponseTransformers"/>
     </bean>
 
+    <reference-list id="listOfMessageValidators" interface="ddf.catalog.validation.MessageValidator"
```

Comment:
If we actually care about the list's order then the `SortedServiceList` bean has to be defined outside of the `reference-listener` and we have to use it as the `ref` in the `MessageValidatorManager` bean. Currently the `reference-list` is being passed to the `MessageValidatorManager` and that list isn't sorted. If the list's order doesn't matter then you can just use the `reference-list` (as you currently are) and delete the `reference-listener`.

Reply:
We do care about the order. Should I delete the `reference-list` and the `reference-listener`, define the `SortedServiceList` as a bean, and pass that bean as the ref in `MessageValidatorManager`?